src/compiler/transformers/es2015.ts

```diff
@@ -1642,12 +1641,12 @@ export function transformES2015(context: TransformationContext): (x: SourceFile
             case SyntaxKind.PropertyDeclaration: {
                 const named = node as AccessorDeclaration | MethodDeclaration | PropertyDeclaration;
                 if (isComputedPropertyName(named.name)) {
-                    return factory.replacePropertyName(named, visitEachChild(named.name, elideUnusedThisCaptureWorker, nullTransformationContext));
+                    return factory.replacePropertyName(named, visitEachChild(named.name, elideUnusedThisCaptureWorker, /*context*/ undefined));

```

Comment:
The uses of `nullTransformationContext` (and now `undefined`) in this file seem suspicious; all of these call sites have a `context` available and using it still passes tests. @rbuckton should I be "fixing" these?